hack/installers/checksums/awscli-linux-arm64-v2.4.6.zip.sha256

```diff
@@ -0,0 +1 @@
+095f5895976155ec570ea550ad81f92b88f1d55f1d7fc8813b30f329a82a83ba  awscli-linux-arm64-v2.4.6.zip
```

Comment:
We would also need s390x & ppc64le binaries.   Last time I checked Amazon did not provide these.   That is one of the reasons we implemented the argocd-K8s-auth while reducing the size of the of the final image.

Reply:
@34fathombelow we’re just using this branch to build a one-off image to narrow down an issue. Might be one of many experiments, ‘cause we’re very short on ideas. 😛

Reply:
I see.. tks for pointing out. At this point, this branch is mainly an exercise to help us investigating a possible issue with `argocd-K8s-auth`. We are getting an elevated number of `Unauthorized` errors in our internal servers.